src/Twig/ParameterExtension.php

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types=1);
+
+namespace StefanDoorn\SyliusGtmEnhancedEcommercePlugin\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class ParameterExtension extends AbstractExtension
+{
+    /** @var array */
+    protected $parameters;
+
+    /**
+     * @param bool $purchases
+     * @param bool $product_impressions
+     * @param bool $product_detail_impressions
+     * @param bool $product_clicks
+     * @param bool $cart
+     * @param array $checkout
+     */
+    public function __construct(
+        bool $purchases,
+        bool $product_impressions,
+        bool $product_detail_impressions,
+        bool $product_clicks,
+        bool $cart,
+        array $checkout
+    )
+    {
+        $this->parameters = [
+            'purchases' => $purchases,
+            'product_impressions' => $product_impressions,
+            'product_detail_impressions' => $product_detail_impressions,
+            'product_clicks' => $product_clicks,
+            'cart' => $cart,
+            'checkout' => $checkout,
+        ];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
+            new TwigFunction('sylius_gtm_enhanced_ecommerce_parameter', array($this, 'getParameter')),
+        );
+    }
+
+    /**
+     * @param string $name
+     * @return bool|array
+     */
+    public function getParameter(string $name)
```

Comment:
Can this one also have a return type?

Reply:
Because there is more than one return type this method could not have a return type